contracts/examples/SimpleInterestTermsContract.sol

```diff
@@ -281,15 +318,16 @@ contract SimpleInterestTermsContract is TermsContract {
 
         // Index of the token used for principal payments in the Token Registry
         uint principalTokenIndex;
-        // Amount, denominated in the aforementioned token, expected in total principal
-        // plus interest.
-        uint principalPlusInterest;
-        // The amortization unit in which the repayments installments schedule is defined
-        uint amortizationUnitTypeAsUint;
+        // The principal amount denominated in the aforementioned token.
+        uint principalAmount;
+        // The loan's interest rate.
+        uint interestRate;
```

Comment:
same comment as above -- let's be explicit a out the fact that the interest rate accrues once per amortization unit, and not ever the _whole_ loan's term.